aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -401,23 +401,23 @@ module AptosFramework::Coin {
         register<FakeMoney>(source);
         register<FakeMoney>(destination);
         let coins_minted = mint<FakeMoney>(amount, &mint_cap);
-        deposit(Signer::address_of(source), coins_minted);
+        deposit(signer::address_of(source), coins_minted);
         move_to(source, FakeMoneyCapabilities {
             mint_cap,
             burn_cap
         });
     }
 
     #[test(source = @0x1, destination = @0x2)]
-    public(script) fun end_to_end(
+    public entry fun end_to_end(
```

Comment:
What's the diff between public(script) and public entry?

Reply:
None, just syntax change. The name "script" for a function is really idiotic, scripts are the thing which we send for execution.